.gitmodules

```diff
@@ -4,3 +4,6 @@
 [submodule "subprojects/libliftoff"]
 	path = subprojects/libliftoff
 	url = https://github.com/emersion/libliftoff.git
+[submodule "subprojects/stb"]
```

Comment:
We have SDL as a mandatory dep already. Does it makes sense to use SDL to load images, instead of adding an additional dep?

Reply:
We were discussing making screenshots write as PNG the other day and using this came up which is why I started using it here.

Reply:
I see, makes sense. WFM!